### ISSUE TITLE
Surveyor sometimes displays wrong Worker ID on thank you page 

### ISSUE THREAD

**Issue by TutiGomoka** [Watts-Lab/surveyor#113]
We've received emails from several workers about this so far. The wrong Worker ID is displayed on thank you page. 
Thankfully this hasn't affected our data/payments so far. i.e. the data is recorded to same person we notified. 
But this may be worth looking into

<img width="1440" alt="Screen Shot 2022-08-09 at 1 24 45 PM" src="https://user-images.githubusercontent.com/35717402/183717995-89fdb089-0145-4e30-be7d-bfaca8da2177.png">

**Comment by markwhiting** [comment]
Is there any sign of an inconsistency in the data? e.g., do we have responses from the user we think they were, do we have a response from the correct user? Are the dates and times accurate? 

(also, screenshots like this can leak a lot of information sometimes (this is a public repo), it might be safer to just explain the problem or paste a snippet of text)

**Comment by TutiGomoka** [comment]
Yes; good point on screen grab; I changed that and deleted the history revision for it. 

Re: the issue, no, I don't see any inconsistency in data. As an example, in today's particular case with Jon:
I have data for a A14GMP7NT3QB13 which is the correct ID, who we notified (I see this in the quoted email). 
I don't have any data for the wrong ID he stated seeing on the final page. 

I can't tell accuracy of dates because I don't know when someone actually started/finished

**Comment by markwhiting** [comment]
OK. Do we have any data bout the other user? Like do we know when each one was notified and last provided a result?

**Comment by TutiGomoka** [comment]
A14GMP7NT3QB13 is Jon was notified today and we got their response today. 

For the wrong ID A11YY75YZQW908; As of now, I don't see a person with that ID when I make a call to our surveyor responses

**Comment by markwhiting** [comment]
OK, interesting. What a strange problem.
